src/common/dialog/AddonDispatchDialog.svelte

```diff
@@ -0,0 +1,144 @@
+<script>
```

Comment:
I'm going to hold off on commenting on this as it is still under heavy development - we can review it more together if needed